treeherder/etl/buildbot.py

```diff
@@ -231,6 +231,14 @@
         }
     },
     {
+        'regex': re.compile(r'WINNT 6\.1|win7|Windows 7 VM.*32-bit', re.IGNORECASE),
```

Comment:
Eg the WINNT and win7 bits will match both the VM and non-VM entries, and which ends up matching will just be based on order